akka-http-tests/src/test/scala/akka/http/scaladsl/server/TestServer.scala

```diff
@@ -21,18 +25,35 @@ object TestServer extends App {
     akka.log-dead-letters = off
     akka.stream.materializer.debug.fuzzing-mode = off
     """)
+
   implicit val system = ActorSystem("ServerTest", testConf)
   import system.dispatcher
   implicit val materializer = ActorMaterializer()
 
+  // --------- json streaming ---------
+  import spray.json.DefaultJsonProtocol._
+  import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
+  final case class Tweet(message: String)
+  implicit val tweetFormat = jsonFormat1(Tweet)
+
+  // FIXME: Need to be able to support composive framing with content type (!!!!!!!)
+  import akka.http.scaladsl.server.EntityStreamingSupport._
+  /* override if extending EntityStreamingSupport */
+  implicit val incomingEntityStreamFraming: FramingWithContentType = bracketCountingJsonFraming(128)
+  /* override if extending EntityStreamingSupport */
+  implicit val outgoingEntityStreamRendering: SourceRenderingMode = JsonSourceRenderingModes.LineByLine
```

Comment:
The second example could maybe just be "FramingSupport" so that one could simply write
```scala
import MyProtocol._
import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
import akka.http.scaladsl.server.FramingSupport

implicit val csvFraming = FramingSupport.withContentType(`text/csv`).withFraming("", "\n", "")
```
to support other media types.